guardian/grid
# upgrade guardian/titip

## What does this change?

Upgrade titip to 1.1.0 (guardian/titip#4) to workaround chrome css selector bugs

## How should a reviewer test this change?

Deploy to TEST. Do tooltips appear correctly (visible only on hover; rounded borders, etc.) to a user who is affected by the bug (ie. tooltips do not appear correctly for them on PROD)

## How can success be measured?

## Who should look at this?


## Tested? Documented?
